FP_Factions/gordon_reskins/RHSak74/config.cpp

```diff
@@ -0,0 +1,115 @@
+class CfgPatches
+{
+	class gordon_reskins_RHSAK74stuff
+	{
+		// List of units defined in this "PBO"
+		units[]={};
+		weapons[]=
+		{
+			"rhs_weap_ak74m_camo_npz",

```

Comment:
Why not have this follow the FP_GOR naming scheme?

Reply:
Because it's not a new class, it's an RHS class I'm editing because it's broken and I want it to be not broken.